### ISSUE TITLE
https://github.com/yxuansu/TaCL/blob/main/pretraining/train.py#L97

### ISSUE THREAD

**Issue by tongshoujie** [yxuansu/TaCL#12]
https://github.com/yxuansu/TaCL/blob/main/pretraining/train.py#L97
为什么需要这一行代码呀？scheduler.step()不就可以了吗？

**Comment by yxuansu** [comment]
> https://github.com/yxuansu/TaCL/blob/main/pretraining/train.py#L97 为什么需要这一行代码呀？scheduler.step()不就可以了吗？

嗯嗯，你的实现方式也可以哈。这只是我写代码的一个个人习惯哈。

**Comment by tongshoujie** [comment]
thanks
